.github/workflows/ci.yml

```diff
@@ -51,15 +52,28 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: bash libyuv.cmd
+    - name: Build googletest
+      # TODO: if: steps.cache-ext.outputs.cache-hit != 'true'
+      working-directory: ./ext
+      run: bash googletest.cmd
+      # Note: "apt install googletest" is sometimes insufficient for find_package(GTest) so build in ext/ instead.
 
     - name: Prepare libavif (cmake)
       run: |
         mkdir build && cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DAVIF_CODEC_AOM=ON -DAVIF_LOCAL_AOM=ON -DAVIF_CODEC_DAV1D=ON -DAVIF_LOCAL_DAV1D=ON -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON -DAVIF_CODEC_SVT=ON -DAVIF_LOCAL_SVT=ON -DAVIF_CODEC_LIBGAV1=ON -DAVIF_LOCAL_LIBGAV1=ON -DAVIF_LOCAL_LIBYUV=ON -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON -DAVIF_BUILD_TESTS=ON
+        cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF \
+          -DAVIF_CODEC_AOM=ON -DAVIF_LOCAL_AOM=ON \
+          -DAVIF_CODEC_DAV1D=ON -DAVIF_LOCAL_DAV1D=ON \
+          -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON \
+          -DAVIF_CODEC_SVT=ON -DAVIF_LOCAL_SVT=ON \
+          -DAVIF_CODEC_LIBGAV1=ON -DAVIF_LOCAL_LIBGAV1=ON \
+          -DAVIF_LOCAL_LIBYUV=ON \
+          -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON \
+          -DAVIF_BUILD_TESTS=ON -DAVIF_BUILD_GTESTS=ON
     - name: Build libavif (make)
       working-directory: ./build
       run: make -j $(($(nproc) + 1))
     - name: Run AVIF Tests (on Linux)
       if: runner.os == 'Linux'
       working-directory: ./build
-      run: ./tests/aviftest ../tests/data
+      run: ./tests/aviftest ../tests/data && ctest -j $(($(nproc) + 1))
```

Comment:
Optional: I suggest we move this change and the other changes I suggested in this comment to a separate pull request, because they are independent of GoogleTest.

Nit: Change `&&` to a semicolon `;`. Alternatively, delete `&&` and move the `ctest -j $(($(nproc) + 1))` command to the next line. If `aviftest` fails, we should still run the other tests.

Note 1: We should also run the `avifyuv` test.

Note 2: We should move `aviftest` and `avifyuv` to the ctest framework (and remove the custom target `avif_test_all`). Then the `ctest -j $(($(nproc) + 1))` command will run all tests.

Reply:
Done at PR https://github.com/AOMediaCodec/libavif/pull/943.